lightning_pose_app/backend/extract_frames.py

```diff
@@ -344,7 +344,7 @@ def export_frames(
     for frame, idx in zip(frames, frame_idxs):
         cv2.imwrite(
             filename=os.path.join(save_dir, "img%s.%s" % (str(idx).zfill(n_digits), format)),
-            img=frame[0],
+            img=cv2.cvtColor(frame.transpose(1, 2, 0), cv2.COLOR_RGB2BGR),
```

Comment:
is the conversion from RGB to BGR necessary here? this should just work if we save in RGB format right?

Reply:
Yes. Im getting an odd images when I remove the conversion. looks like one of the channels is missing..